source/MRViewer/MRSceneObjectsListDrawer.h

```diff
@@ -0,0 +1,103 @@
+#pragma once
+#include "exports.h"
+
+namespace MR
+{
+
+class Object;
+
+class MRVIEWER_CLASS SceneObjectsListDrawer
+{
+public:
+    MRVIEWER_API SceneObjectsListDrawer();
+    MRVIEWER_API virtual ~SceneObjectsListDrawer() = default;
```

Comment:
no need to `MRVIEWER_API` for default function

Reply:
fixed